src/JSInjection/Modal/ReadArticle.js

```diff
@@ -41,15 +41,15 @@ export function ReadArticle({
   return (
     <>
       <div className="article-container">
-        <StyledSmallButtonBlue onClick={reportProblem}>
+        {/* <StyledSmallButtonBlue onClick={reportProblem}>
```

Comment:
why do we commit commented out code?

Reply:
This must have been a mistake when I was trying to resolve conflicts and change the origin of the branch